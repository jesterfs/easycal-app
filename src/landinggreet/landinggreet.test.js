import React from 'react'
import ReactDOM from 'react-dom'
import { BrowserRouter } from 'react-router-dom'
import LandingGreet from './landinggreet'
import renderer from 'react-test-renderer';

describe('LandingGreet component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(
      <BrowserRouter>
        <LandingGreet />
      </BrowserRouter>,
      div
    )
    ReactDOM.unmountComponentAtNode(div)
  })
  
  it('renders the UI as expected', () => {
    const div = document.createElement('div')
    const tree = renderer
      .create(<BrowserRouter>
        <LandingGreet />
      </BrowserRouter>,
      div)
      .toJSON();
    expect(tree).toMatchSnapshot();  
    });

});