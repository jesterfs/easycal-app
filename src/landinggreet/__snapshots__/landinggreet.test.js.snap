// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingGreet component renders the UI as expected 1`] = `
<div
  className="LandingGreet, greetgroup"
>
  <div
    className="item"
  >
    <h2>
      Meet EasyCal
    </h2>
  </div>
  <div
    className="item"
  >
    <h3>
      A scheduling solution for the modern office.
    </h3>
  </div>
  <div>
    <a
      href="/signup"
      onClick={[Function]}
    >
      <button>
        Create an Account
      </button>
    </a>
  </div>
</div>
`;
